Example/WebIOPi/PinTableViewController.swift

```diff
@@ -183,14 +183,20 @@ private extension PinTableViewController {
         }
 
         var sequence = [Value]()
-        for index in sequenceText.characters.indices {
-            let character = sequenceText[index]
-            let string = String(character)
-
+        for (index, element) in sequenceText.enumerated() {
```

Comment:
The `index` parameter isn't used here, so let's replace it with `_` to suppress the warning.